Box_intersection_d/doc/Box_intersection_d/dependencies

```diff
@@ -4,3 +4,4 @@ STL_Extension
 Algebraic_foundations
 Circulator
 Stream_support
+Polyhedron
```

Comment:
I guess this shouldn't have been changed, right?

Reply:
In order to get a [link ](https://doc.cgal.org/latest/Box_intersection_d/index.html#title4) here in the User Manual